pkg/fanal/analyzer/language/elixir/mixlock/mixlock.go

```diff
@@ -0,0 +1,48 @@
+package mixlock
+
+import (
+	"context"
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/hex/mix"
+
+	"golang.org/x/xerrors"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&mixLockAnalyzer{})
+}
+
+const (
+	version        = 1
+	fileNameSuffix = "mix.lock"
```

Comment:
Can it be a suffix? Does it mean `myproject-mix.lock` or something like that can be used?

Reply:
Also, it is better to define the file name here.
https://github.com/aquasecurity/trivy/blob/55cb4ad2f158beec9a5f9dd92ef004ad24d04165/pkg/fanal/types/const.go#L55-L76

Reply:
you are right. Thanks. I moved this to `trivy/pkg/fanal/types/const.go`